impl/sql/testdb/testdb_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testdb
+
+import (
+	"context"
+	"testing"
+)
+
+func TestNewForTest(t *testing.T) {
+	ctx := context.Background()
+	db, done := NewForTest(ctx, t)
```

Comment:
I wonder if you could also test that `db.Ping()` fails after `done` is called.

Reply:
`db.Ping()` succeeds afterwards because the connection to MySQL is still alive.

Reply:
SGTM. Any other easy way to test that the DB was dropped?

Reply:
E.g. call `ktsql.Open()` again.

Reply:
The `done(ctx)` function now returns an error that we can check